src/aggregations/pipeline-aggregations/bucket-sort-aggregation.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+const PipelineAggregationBase = require('./pipeline-aggregation-base');
+
+const ES_REF_URL =
+    'https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-pipeline-bucket-sort-aggregation.html';
+
+/**
+ * A parent pipeline aggregation which sorts the buckets of its parent
+ * multi-bucket aggregation. Zero or more sort fields may be specified
+ * together with the corresponding sort order. Each bucket may be sorted
+ * based on its _key, _count or its sub-aggregations. In addition, parameters
+ * from and size may be set in order to truncate the result buckets.
+ *
+ * [Elasticsearch reference](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-pipeline-bucket-sort-aggregation.html)
+ *
+ * @example
+ * const reqBody = esb.requestBodySearch()
+ *     .agg(
+ *         esb.bucketSortAggregation('sort')
+ *             .sort([
+ *                  esb.sort('user', 'desc')
+ *              ])
+ *              .from(5)
+ *              .size(10)
+ *         )
+ *     );
+ *
+ * @param {string} name The name which will be used to refer to this aggregation.
+ *
+ * @extends PipelineAggregationBase
+ */
+class BucketSortAggregation extends PipelineAggregationBase {
+    // eslint-disable-next-line require-jsdoc
+    constructor(name) {
+        super(name, 'bucket_sort', ES_REF_URL);
+    }
+
+    /**
+     * Sets the list of fields to sort on. Optional.
+     *
+     * [Elasticsearch reference](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-pipeline-bucket-sort-aggregation.html#id-1.9.5.55.4.6)
```

Comment:
I don't think it is necessary to repeat the URL for reference and example in every method's docs. Could you please remove them?